src/docfx/lib/markdown/validation/ContentValidationExtension.cs

```diff
@@ -33,9 +33,8 @@ internal static class ContentValidationExtension
                                 Level = headingBlock.Level,
                                 SourceInfo = headingBlock.ToSourceInfo(),
                                 Content = GetHeadingContent(headingBlock), // used for reporting
-
-                                // HeadingChar = headingBlock.HeadingChar
-                                // RenderedPlainText = MarkdigUtility.ToPlainText(headingBlock), // used for validation
+                                HeadingChar = headingBlock.HeaderChar,
+                                RenderedPlainText = MarkdigUtility.ToPlainText(headingBlock), // used for validation
```

Comment:
What's the difference between `Content` and `RenderedPlainText`?

Reply:
offline talked, content is the user input, rendered plain text is the build output. the previous one is for reporting, the later one is for validation.